automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala

```diff
@@ -598,6 +598,8 @@ class AuthDomainSpec extends FreeSpec /*with ParallelTestExecution*/ with Matche
 
             api.billing.removeUserFromBillingProject(projectName, user.email, BillingProjectRole.Owner)
             checkNoAccess(user, projectName, workspaceName)
+
+            api.groups.removeUserFromGroup(groupName, user.email, GroupRole.Member)
```

Comment:
These should be:
```
register cleanUp api.groups.removeUserFromGroup(groupName, user.email, GroupRole.Member)
```
and be done immediately after the call to `addUserToGroup`. You may also need to make use of `withCleanUp` (inside the `withGroup`) to make sure `removeUserFromGroup` happens before the group is deleted. I'm not sure how Google handles removing users from groups that have been deleted.

I'd be surprised if deleting the group doesn't also effectively wipe it from the list of groups a user belongs to. In fact, I felt a little silly removing members before deleting the group in `withGroup` but figured it didn't hurt.

Sorry for missing this clean-up.